### PR TITLE
[MISC] Radiology Image Compression

### PR DESCRIPTION
**Pull request by Rakshith2597**
PR contains 
1. Codes for radiology image compression published in [Designing Deep Neural High-Density Compression Engines for Radiology Images](https://link.springer.com/article/10.1007/s00034-022-02222-0), [Circuits, Systems, and Signal Processing](https://link.springer.com/journal/34) (2022) by Springer Nature.
2. Update in URLs for accessing models in chest_xray_screening and mammogram_screening.

I am attaching a screenshot of pytest output obtained on our server for reference
miriad_rad_compress_unittest

This is part of the project MIRIAD: Many Incarnations of Screening of Radiology for High Throughput Disease Screening via Multiple Instance Reinforcement Learning with Adversarial Deep Neural Networks, sponsored by INTEL TECHNOLOGY INDIA PVT. LTD.

Principal Investigators:

Dr Debdoot Sheet (PI), Dr Nirmalya Ghosh (Co-PI)
Department of Electrical Engineering
Indian Institute of Technology Kharagpur

Dr Ramanathan Sethuraman (Co-PI)
Intel Technology India Pvt. Ltd.

### CODE REVIEW

**Review by goodsong81 — CHANGES_REQUESTED** [review]
It looks good to me in overall. But could you resolve 1 critical issue from static code analysis?
https://app.codacy.com/gh/openvinotoolkit/training_extensions/file/79764203338/issues/source?bid=30977325&fileBranchId=32430119#l38

**Comment by Rakshith2597** [comment]
> It looks good to me in overall. But could you resolve 1 critical issue from static code analysis? https://app.codacy.com/gh/openvinotoolkit/training_extensions/file/79764203338/issues/source?bid=30977325&fileBranchId=32430119#l38

@goodsong81 
This line of code is used to convert the ONNX model to OpenvinoIR (following [this](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html#doxid-openvino-docs-m-o-d-g-deep-learning-model-optimizer-dev-guide) documentation). All models in the misc branch uses this to convert the model. I have used [this](https://github.com/openvinotoolkit/training_extensions/blob/misc/misc/pytorch_toolkit/text_recognition/text_recognition/utils/exporter.py) file as the template for all our models. For now, I have replaced the subprocess call with os.system, but I doubt it would resolve the issue showed in codacy. Please let me know if there is an alternate way to create the IR using Model Optimiser.

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ data/*
 
 *.tar.gz
 *.zip
+misc/pytorch_toolkit/miriad_compression_codes/model_weights/*
```
Comment (goodsong81) [inline .gitignore]: Duplicate w/ misc/pytorch_toolkit/radiology_compression/.gitignore

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Security issue was resolved. Thank you.
Could you adjust a tiny bit more?

File: .gitignore (hunk 1 above)
Comment (Rakshith2597) [reply .gitignore]: Removed it.

**Review by goodsong81 — APPROVED** [review]
LGTM.

**Pull request merged**